spec/factories/apply_applications.rb

```diff
@@ -5,14 +5,33 @@
     sequence(:apply_id)
     application { ApiStubs::ApplyApi.application }
     invalid_data { {} }
-    provider
+    provider_code { create(:provider).code }
```

Comment:
Could this be a trait to be invoked explicitly? Would we need to care about / do something about applications where providers do not exist yet?

Reply:
I think every application has a provider code, but we might get the odd application without one - who knows. We'll find out when this goes live.